third_party/2and3/jinja2/environment.pyi

```diff
@@ -97,6 +98,13 @@ class Template:
     @property
     def debug_info(self): ...
 
+    if sys.version_info >= (3, 6):
+        from typing import Awaitable, AsyncIterator
```

Comment:
Could you put these imports in a separate `if sys.version_info` block at the top? An import statement within a class seems, um, unusual.